### PR TITLE
OTS stuff

### PR DESCRIPTION
**Pull request by anandhs29**

### CODE REVIEW

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class CompaniesController < ApplicationController
+    before_filter :authenticate, :except=>[:company_by_pin, :new]
```
Comment (anandhs29) [inline app/controllers/companies_controller.rb]: spacing is weird here (only in the diff), but what is the coding standard for rails? spaces or tabs. The file didn't follow any.

File: db/seeds.rb [hunk 2]
```diff
@@ -61,11 +61,6 @@
 @apiKey.access_token = '7a09be3f769059d69d04068c8fcdb3b9'
 @apiKey.save
 
-@zenefits = Company.create({name: 'Zenefits'})
-@amazon =  Company.create({name: 'Amazon'})
-@xe = Company.create({name: 'XE'})
-@fresh = Company.create({name: 'FreshBooks'})
```
Comment (ragulanr) [inline db/seeds.rb]: @anandhs29 why are you deleting the companies?

Comment (anandhs29) [inline db/seeds.rb]: You're right. It is not necessary.

**Comment by ragulanr** [comment]
lgtm

**Pull request merged**